packages/@react-aria/select/src/HiddenSelect.tsx

```diff
@@ -87,7 +87,8 @@ export function useHiddenSelect<T>(props: AriaHiddenSelectProps, state: SelectSt
       name,
       size: state.collection.size,
       value: state.selectedKey ?? '',
-      onChange: (e: React.ChangeEvent<HTMLSelectElement>) => state.setSelectedKey(e.target.value)
+      onChange: (e: React.ChangeEvent<HTMLSelectElement>) => state.setSelectedKey(e.target.value),
+      ['data-rsp-a11y-ignore']: true
```

Comment:
Think we can get rid of this one too. I figure people can blanket filter out the container div for aXe and pa11y via `data-rsp-a11y-ignore` (its the one triggering the "ARIA hidden element must not be focusable or contain focusable elements" false positive) and thus we don't need the attribute on the select props. We could even move this line onto the `containerProps` so people using the hook instead of HiddenSelect benefit too.

Did you find that you needed the data attribute on the select element?